dist/frontAPImanager.js

```diff
@@ -0,0 +1,16 @@
+class APImanagerFront {
```

Comment:
try to add the .gitignore file before you commit to github. otherwise it does not work